python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -271,6 +276,7 @@ def _output_ready_callback(task_index, output: RefBundle):
             self._metrics.cur += allocated
             if self._metrics.cur > self._metrics.peak:
                 self._metrics.peak = self._metrics.cur
+            self._task_blocks_outputted[task_index] += 1
```

Comment:
Might be more readable. It wasn't immediately obvious to me why we're adding by 1, and not the number of blocks
```suggestion
            self._task_blocks_outputted[task_index] += len(output.blocks)
```